extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -1837,5 +1840,40 @@ describes.realWin(
         ).to.be.false;
       });
     });
+
+    describe('page loading', () => {
+      it('should load the active page on layout', async () => {
+        const pages = await createStoryWithPages(
+          2,
+          ['page-1', 'page-2'],
+          false
+        );
+        env.sandbox.stub(story, 'mutateElement').callsFake((mutator) => {
+          mutator();
+          return Promise.resolve();
+        });
+        await story.buildCallback();
+        await story.layoutCallback();
+        expect(pages[0].hasAttribute('distance')).to.be.true;
+      });
+
+      it('should load the inactive pages after the active page is loaded', async () => {
```

Comment:
ditto, it just tests the positioning, which is fine but plz be accurate in your descriptions